AI #2334: Align remaining normative requirement inconsistencies with updated guidelines (#2344)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/datasets/billing_period/columns/billingperiodstatus.md

* BillingPeriodStatus MUST NOT be null.
* BillingPeriodStatus MUST be one of the allowed values.
* BillingPeriodStatus MUST represent the state of the billing period identified by [BillingPeriodStart](#datasets.billingperiod.billingperiodstart) and [BillingPeriodEnd](#datasets.billingperiod.billingperiodend).

> * BillingPeriodStatus MUST NOT be "Open" following a "Closed" state, except when explicitly requested or approved by the customer.

Author: ```suggestion
* BillingPeriodStatus MUST NOT be "Open" following a previous status of "Closed", except when explicitly requested or approved by the customer.
```

> * BillingPeriodStatus MUST NOT be "Open" when BillingPeriodStatus was previously "Closed", unless explicitly requested or approved by the customer.

Reviewer: A bit smoother:

```suggestion
* BillingPeriodStatus MUST NOT be "Open" following a "Closed" state, except when explicitly requested or approved by the customer.
```
